src/io/flutter/view/InspectorPanel.java

```diff
@@ -45,6 +47,9 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+// TODO(devoncarew): Should we filter out the CheckedModeBanner node type?
+// TODO(devoncarew): Should we filter out the WidgetInspector node type?
```

Comment:
👍

Reply:
sgtm. 
Filtering out the WidgetInspector shouldn't be needed at some point once an inversion of control occurs on the package:flutter side but it is reasonable to filter it for now. Question is though, what happens if you trigger an inspect action on a widget under the WidgetInspector which can currently happen.